src/twilio_util.ts

```diff
@@ -57,5 +57,5 @@ export function formatAttachments(reqBody: TwilioRequestBody): string {
     .map((url, i) => `<${url}|Attachment ${i + 1}>`)
     .join(' ');
 
-  return `${reqBody.Body} [This message included attachments: ${mediaURLsFormatted}]`;
+  return `${reqBody.Body}\n[${mediaURLsFormatted}]`;
```

Comment:
I think the [This message included attachments:] bit helps add clarity as to where the link came from (ie to differentiate from the user sending a link themselves).